docs/installation.md

```diff
@@ -81,6 +81,15 @@ $ npm start
 
 7. open `index.html` in a browser
 
+Optionally, you can define Mithril as a Webpack plugin, the benefit is that Mithril will be included globally and you will not have have to `import m from "mithril"`:
+```json
+plugins: [
+    new webpack.ProvidePlugin({m: 'mithril'}),
+    // ...
+]
+```
+Then you could remove the import line from step 4 (don't forget to restart Webpack if you ran it with `--watch`) and it will work just the same.
```

Comment:
```suggestion
Then, you could remove the import line from step 4 (don't forget to restart Webpack if you ran it with `--watch`), and it will work just the same.
```